src/date-range-picker/__tests__/date-range-picker-absolute.test.tsx

```diff
@@ -168,6 +169,21 @@ describe('Date range picker', () => {
         expect(wrapper.findDropdown()!.findStartTimeInput()!.findNativeInput().getElement()).toHaveValue('00:00:00');
       });
 
+      test('start date can be selected with keyboard', () => {
+        const { wrapper } = renderDateRangePicker({
+          ...defaultProps,
+          value: { type: 'absolute', startDate: '2021-03-02T05:00:00+08:45', endDate: '2021-03-12T13:05:21+08:45' },
+        });
+        wrapper.findTrigger().click();
+
+        wrapper.findDropdown()!.findDateAt('left', 3, 4).focus();
```

Comment:
Is this line necessary or would `keydown` work all the same without focusing first?

Reply:
You need to focus first!